react/tic-tac-toe/src/redux/users/actions.js

```diff
@@ -22,6 +22,11 @@ const actionsCreators = {
     } else {
       dispatch({ type: actions.LOGIN_FAILURE });
     }
+  },
+  logout: () => dispatch => {
+    localStorage.removeItem('token');
+    api.setHeaders(defaultHeaders);
+    dispatch(push('/'));
```

Comment:
Same as previous PR. Clean the storage and your store too. Use redux to hold the token to keep consistency of your data flow.